### PR TITLE
Dap 3753

### PR DESCRIPTION
**Pull request by Lisofffa**
#### What is the purpose of this pull request? (put an "X" next to an item)

[ ] Bug fix
[ ] New feature
[ ] Documentation update
[ ] Add something to the core
[ ] Other, please explain:

#### Add a reference to a related issue if it exists

*issue:*

### CODE REVIEW

File: src/contentscript/overlay/root/components/OverlayToolbar/Notification/index.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  if (payload && !payload.payload) {
+    setTimeout(() => {
+      notificationRef.current?.classList.add('remove_notification')
+    }, 9500)
+    setTimeout(() => {
+      setPinnedNotification(false)
+
+      onRemove(payload)
+    }, 10000)
+  }
```
Comment (alsakhaev) [inline src/contentscript/overlay/root/components/OverlayToolbar/Notification/index.tsx:47]: Move to useEffect and don't forget to unsubscribe (clearTimeout)

**Review by alsakhaev — CHANGES_REQUESTED** [review]

**Comment by Gromazek** [comment]
test

**Comment by Gromazek** [comment]
test

**Pull request merged**